pkg/blockstorage/vmware/vmware.go

```diff
@@ -216,6 +216,8 @@ func (p *FcdProvider) SnapshotCreate(ctx context.Context, volume blockstorage.Vo
 	if snap.Size == 0 {
 		snap.Size = volume.Size
 	}
+
+	snap.Volume = &volume
```

Comment:
Since this is an expected invariant of the API contract, it makes sense to add a unit test for it.

Reply:
@redgoat650 I know, no good deed goes unpunished. Sorry.